test/src/tests-user.ts

```diff
@@ -134,23 +123,24 @@ describe('mParticle User', function() {
                 .Identity.getCurrentUser()
                 .setConsentState(consentState);
 
-            mParticle._resetForTests(MPConfig, true);
-            mParticle.init(apiKey, mParticle.config);
```

Comment:
Why are reset and init being removed?

Reply:
this was not necessary to begin with. I don  tknow why we had it.  we should only be doing a `resetForTests` in the beginning of a test, not in the middle